### PR TITLE
Kirill.sitcevoy/systemconsole

### PR DESCRIPTION
**Pull request by kievzenit**
System.Console home task

### CODE REVIEW

**Review by horianyn — DISMISSED** [review]

File: Hello_Console_stud/Hello_Console_stud/Program.cs [hunk 1]
```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Threading;
+
+namespace Hello_Console_stud
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a;
```
Comment (Artya) [inline Hello_Console_stud/Hello_Console_stud/Program.cs]: what is mean `a`?

File: Hello_Console_stud/Hello_Console_stud/Program.cs [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                    4.  morse sos
+          
+                    ");
+                    try
+                    {
+                        a = (int)uint.Parse(Console.ReadLine());
+                        switch (a)
+                        {
+                            case 1:
+                                My_strings();
```
Comment (Artya) [inline Hello_Console_stud/Hello_Console_stud/Program.cs]: Should be MyString()

File: Hello_Console_stud/Hello_Console_stud/Program.cs [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+                                Console.WriteLine("");
+                                break;
+                            case 2:
+                                My_Binary();
```
Comment (Artya) [inline Hello_Console_stud/Hello_Console_stud/Program.cs]: Should be MyBinary()

File: Hello_Console_stud/Hello_Console_stud/Program.cs [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+                                Console.WriteLine("");
+                                break;
+                            case 3:
+                                My_music();
```
Comment (Artya) [inline Hello_Console_stud/Hello_Console_stud/Program.cs]: Should be MyMusic()

File: Hello_Console_stud/Hello_Console_stud/Program.cs [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+                                Console.WriteLine("");
+                                break;
+                            case 4:
+                                Morse_code();
```
Comment (Artya) [inline Hello_Console_stud/Hello_Console_stud/Program.cs]: Should be MorseCode()

File: Hello_Console_stud/Hello_Console_stud/Program.cs [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+        static void Morse_code()
+        {
+            //Create string variable for 'sos'      
+            var sos = "sos";
+            //Use string array for Morse code
+            string[,] Dictionary_arr = new string [,] { { "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9" },
+            { ".-   ", "-... ", "-.-. ", "-..  ", ".    ", "..-. ", "--.  ", ".... ", "..   ", ".--- ", "-.-  ", ".-.. ", "--   ", "-.   ", "---  ", ".--. ", "--.- ", ".-.  ", "...  ", "-    ", "..-  ", "...- ", ".--  ", "-..- ", "-.-- ", "--.. ", "-----", ".----", "..---", "...--", "....-", ".....", "-....", "--...", "---..", "----." }};
+            //Use ToCharArray() method for string to copy charecters to Unicode character array
+            //Use foreach loop for character array in which
+            foreach (var c in sos.ToCharArray())
```
Comment (Artya) [inline Hello_Console_stud/Hello_Console_stud/Program.cs]: what `c` this mean?

File: Hello_Console_stud/Hello_Console_stud/Program.cs [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+            {
+                for (var i = 0; i < Dictionary_arr.GetLength(1); i++)
+                {
+                    if (c == char.Parse(Dictionary_arr[0, i]))
+                    {
+                        foreach (var s in Dictionary_arr[1, i].ToCharArray())
```
Comment (Artya) [inline Hello_Console_stud/Hello_Console_stud/Program.cs]: what `s` this mean?

File: Hello_Console_stud/Hello_Console_stud/Program.cs (hunk 1 above)
Comment (kievzenit) [reply Hello_Console_stud/Hello_Console_stud/Program.cs]: Fixed in commit 592bca63f86e969b28bb074d314dd91abb86c457

File: Hello_Console_stud/Hello_Console_stud/Program.cs (hunk 2 above)
Comment (kievzenit) [reply Hello_Console_stud/Hello_Console_stud/Program.cs]: This code style for naming methods was in template

Comment (kievzenit) [reply Hello_Console_stud/Hello_Console_stud/Program.cs]: But I fixed this in commit 592bca63f86e969b28bb074d314dd91abb86c457

File: Hello_Console_stud/Hello_Console_stud/Program.cs (hunk 6 above)
Comment (kievzenit) [reply Hello_Console_stud/Hello_Console_stud/Program.cs]: Fixed in commit 592bca63f86e969b28bb074d314dd91abb86c457

File: Hello_Console_stud/Hello_Console_stud/Program.cs (hunk 7 above)
Comment (kievzenit) [reply Hello_Console_stud/Hello_Console_stud/Program.cs]: Fixed in commit 592bca63f86e969b28bb074d314dd91abb86c457

**Review by SergiySimchuk — APPROVED** [review]
I think it have to work)

**Review by horianyn — APPROVED** [review]

**Pull request merged**